client/lib/plugins/notices.jsx

```diff
@@ -763,6 +758,10 @@ export default {
 						);
 				}
 				break;
+			case 'RECEIVE_PLUGINS':
+				return this.props.translate( 'Error fetching plugins on %(numberOfSites)d sites.', {
```

Comment:
Please add plural handling here. Thanks!

Reply:
This line is executed only when `numberOfSites > 1`. The singular case is handled on line 974, in the `singleErrorMessage` function.

Reply:
@akirk For future refactorings, is it possible to merge both cases into one call, isn't it?
```js
translate(
  'Error fetching plugins on %(site)s',
  'Error fetching plugins on %(siteCount)d',
  {
    count: sites.length,
    args: {
      siteCount: sites.length,
      site: sites[0].name,
	}
  }
);
```

Reply:
Please don't do this. Japanese for example doesn't have plurals, so it will always use the first case, even if you have a count > 1.

Reply:
Also, you cannot keep it the way it is because it cannot handle different plural phrases for other languages. For example Russian has different plurals for (2, 3 and 4) and (5, 6).

Too late, it's merged.

Reply:
Thanks for pointing these subtle bugs out, @akirk! This plurals bug in log-store notices was there forever since Calypso was first released, and in this PR, I just moved some code to a different location, without really changing it.

It' hard to fix without an almost complete rewrite. Which will fortunately happen soon as we convert the `log-store` stores to Redux.

Reply:
Alright, no problem. I just caught sight of it here and it's a fair point that you shouldn't be blamed for errors in code you moved :) All good!